src/js/experimental/media-chrome-menu-button.js

```diff
@@ -165,42 +127,37 @@ class MediaChromeMenuButton extends window.HTMLElement {
     // if the menu is hidden, skip updating the menu position
     if (this.#listbox.offsetWidth === 0) return;
 
-    const padding = window.getComputedStyle(this).getPropertyValue('--media-control-padding') || 10;
-
     const buttonRect = this.#button.getBoundingClientRect();
 
-
     // if we're outside of the controller,
     // one of the components should have a media-controller attribute
     // or the button wasn't slotted
     if (
       this.hasAttribute('media-controller') ||
       this.#button.hasAttribute('media-controller') ||
       this.#listbox.hasAttribute('media-controller') ||
-      !this.#buttonSlotted) {
+      !this.#buttonSlotted
+    ) {
       this.#listbox.style.zIndex = '1';
       this.#listbox.style.bottom = 'unset';
-      this.#listbox.style.top = (padding * 2 + buttonRect.height) + 'px'
-      this.#listbox.style.translate = `-${padding}px`;
+      this.#listbox.style.top = buttonRect.height + 'px'
       return;
     }
 
-    const leftOffset = buttonRect.x;
-
-    // Get the element that enforces the bounds for the time range boxes.
+    // Get the element that enforces the bounds for the list boxes.
     const bounds =
       (this.getAttribute('bounds')
         ? closestComposedNode(this, `#${this.getAttribute('bounds')}`)
         : this.parentElement) ?? this;
-    let parentOffset = bounds.getBoundingClientRect().x;
 
-    if (this.#listbox.offsetWidth + leftOffset - parentOffset > bounds.offsetWidth) {
-      const offset = this.#listbox.offsetWidth + leftOffset + buttonRect.width;
-      const overflow = bounds.offsetWidth + parentOffset + (padding * 2) - offset;
-      this.#listbox.style.translate = overflow + 'px';
-    } else {
-      this.#listbox.style.translate = `-${buttonRect.width}px`;
-    }
+    const boundsRect = bounds.getBoundingClientRect();
+    const listboxRect = this.#listbox.getBoundingClientRect();
+    const offset = (listboxRect.width - buttonRect.width) / 2;
+    let position = -offset;
+    position -= Math.max(buttonRect.x + offset + buttonRect.width - boundsRect.right, 0);
+    position += Math.max(-buttonRect.x + offset + boundsRect.x, 0);
```

Comment:
should be able to simplify to
```suggestion
    let position -= Math.max(buttonRect.x + listboxRect.width - boundsRect.right, 0);

```